lib/takeaway.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'menu'
+require 'twilio-ruby'
+
+class Takeaway
+  attr_reader :menu, :basket, :total
+
+  def initialize(menu_klass = Menu.new)
+    @menu = menu_klass
+    @basket = Hash.new(0)
+  end
+
+  def open_menu
+    menu.dishes.map { |item, price| "#{item}: #{'£%.2f' % price}" }
+  end
+
+  def order(item, quantity = 1)
+    basket[item] += quantity
+    "#{quantity}x #{item}(s) added to your basket."
+  end
+
+  def basket_summary
+    basket.map { |item, quantity| "#{item} x#{quantity} = #{'£%.2f' % (menu.dishes[item] * quantity)}" }.join(', ')
```

Comment:
Metrics/LineLength: Line is too long. [115/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/FormatString: Favor `format` over `String#%`. (https://github.com/bbatsov/ruby-style-guide#sprintf)